sos/plugins/rhv_log_collector_analyzer.py

```diff
@@ -35,7 +29,7 @@ def setup(self):
             " --html={report}".format(report=report)
         )
 
-        self.get_cmd_output_now(
+        self.add_cmd_output(
```

Comment:
This still needs to be done for the `rhv-log-collector-analyzer --live --html...` version. You're not even keeping the return value so it's entirely useless to use `get_cmd_output_now()` here; the directory returned by `get_cmd_output_path()` will still be valid for async collection.